app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person_data.get("name"), person_data.get("age"))
+        for person_data in people
+    ]
+
+    for person_data in people:
+        person = Person.people[person_data.get("name")]
+        if "wife" in person_data and person_data.get("wife") is not None:
+            person.wife = Person.people[person_data.get("wife")]
+        if "husband" in person_data and person_data.get("husband") is not None:
+            person.husband = Person.people[person_data.get("husband")]
```

Comment:
You can just write if person_data.get(“wife”) and it will still work. If the key doesn't exist it returns None.
And you don't need to check if "husband" is in person_data because you check it with get.

Reply:
agree, checking `and person_data.get("husband") is not None` is unnecessary